src/Runtime/pdf3/registry.sh

```
@@ -0,0 +1,58 @@
+#!/bin/sh
+set -o errexit
+
+CONTAINER_CMD="";
+
+if command -v docker &> /dev/null; then
```

Shellcheck (0.11.0)

[warning] In POSIX sh, &> is undefined.

(SC3020)